gitops-v2/new/push_aws.yaml

```diff
@@ -0,0 +1,29 @@
+parameters:
+  - name: awsRegion
+    type: string
+    default: ""
+  - name: awsServiceConnection
+    type: string
+    default: ""
+
+steps:
+  - task: AmazonWebServices.aws-vsts-tools.AWSShellScript.AWSShellScript@1
+    displayName: "Get ECR login info"
+    inputs:
+      workingDirectory: "."
+      awsCredentials: ${{ parameters.awsServiceConnection }}
+      regionName: ${{ parameters.awsRegion }}
+      scriptType: inline
+      inlineScript: |
+        set -e
+        SHA_SHORT=$(shaShort)
+
+        aws ecr describe-repositories --repository-names ${SERVICE_NAME} || aws ecr create-repository --repository-name ${SERVICE_NAME} --image-scanning-configuration scanOnPush=true
```

Comment:
Is it really a good idea to auto-create repos? It seems a big exposure for a small gain. In my opinion, the repo should be part of the application's IaC. I'm even skeptical to granting `describe` perms to Azure as that allows the craxor to discover new attack surfaces.

Reply:
This should be handled in separate PR, only moving AWS to make it possible to run without the extension.